### DISCUSSION TITLE
Can I use dynamoDB if I want to host it on AWS

### DISCUSSION THREAD

**Discussion by mosesliao** [opengovsg/FormSG#7980] [Q&A] [answered]
Hi I noticed this app uses nosql DBs so I would like to ask if I host this on AWS can I use dynamoDB to store the data

**Comment by KenLSM** [comment, marked as answer]
DynamoDB has an item size limit of 400kb which may not be sufficient enough for FormSG as we can store datasets that are larger than 400kb.
